invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/bulk_actions/SearchResultsBulkActions.js

```diff
@@ -64,11 +67,13 @@ export class SearchResultsBulkActions extends Component {
             className="align-self-center"
             text={`${selectedCount} ${i18next.t("members selected")}`}
             options={dropdownOptions}
+            aria-label="bulk actions"
```

Comment:
```suggestion
            aria-label="select all members"
```
I guess all ARIA lbls should be translated too?